tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Debugger/ProbesTests.cs

```diff
@@ -362,6 +362,7 @@ private void SetDebuggerEnvironment()
         SetEnvironmentVariable(ConfigurationKeys.Debugger.Enabled, "1");
         SetEnvironmentVariable(ConfigurationKeys.Debugger.MaxDepthToSerialize, "3");
         SetEnvironmentVariable(ConfigurationKeys.Debugger.DiagnosticsInterval, "1");
+        SetEnvironmentVariable(ConfigurationKeys.Debugger.MaxTimeToSerialize, "1000");
```

Comment:
is it because it s slower in CI and times out more?

Reply:
The default time to serialize is 150ms. If we meet this threshold then we stop cloning an object (thus in the resulting snapshot there will be fewer values). In our smoke tests we need to have consistent snapshots, thus I have increased the time to serialize to 1000ms so we will never meet this threshold in our smoke tests.
Does that make sense?